docs/spark-queries.md

```diff
@@ -227,6 +227,20 @@ SELECT * FROM prod.db.table.history
 **This shows a commit that was rolled back.** The example has two snapshots with the same parent, and one is *not* an ancestor of the current table state.
 {{< /hint >}}
 
+### Metadata Logs
+
+To show table metadata logs:
+
+```sql
+SELECT * from prod.db.table.metadata_logs
```

Comment:
Can we rename this metadata table? The name `metadata_logs` doesn't make sense to me since there is just the one log. The rows of the table are log entries.

Reply:
sure, should i then rename it to metadata_log_entries ?